benchmarks/src/main/java/io/grpc/benchmarks/Utils.java

```diff
@@ -252,7 +253,7 @@ public static SimpleResponse makeResponse(SimpleRequest request) {
         throw Status.INTERNAL.augmentDescription("Error creating payload.").asRuntimeException();
       }
 
-      ByteString body = ByteString.copyFrom(new byte[request.getResponseSize()]);
+      ByteString body = UnsafeByteOperations.unsafeWrap(new byte[request.getResponseSize()]);
```

Comment:
Sure I can revert it. I just thought it might help to isolate the actual grpc parts within the benchmarks, assuming that they're intended more for relative measurements than simulating a particular real-world use case.